### PR TITLE
1551 change connection logic in driver to wait to get notification that driver registered

### PR DESCRIPTION
**Pull request by RobertPoienar**

### CODE REVIEW

**Comment by RobertPoienar** [comment]
@alttester/developers

File: Assets/AltTester/Runtime/AltDriver/Communication/DriverWebSocketClient.cs [hunk 1]
```diff
@@ -178,13 +188,14 @@ public void Connect()
...
             }
-
             logger.Debug(string.Format("Connected to: '{0}'.", this.uri));
+
         }
 
         public void Close()
         {
-            logger.Info(string.Format("Closing connection to AltTester® Server on: '{0}'.", this.uri));
+            logger.Info(string.Format("Closing connection to AltServer on: '{0}'.", this.uri));
```
Comment (andreion1ca) [inline Assets/AltTester/Runtime/AltDriver/Communication/DriverWebSocketClient.cs:197]: We should change AltServer to AltTester® Server

**Review by andreion1ca — CHANGES_REQUESTED** [review]

**Review by gomboskriszta15 — APPROVED** [review]

File: Assets/AltTester/Runtime/AltDriver/Communication/DriverWebSocketClient.cs [hunk 2]
```diff
@@ -178,13 +188,14 @@ public void Connect()
...
             }
-
             logger.Debug(string.Format("Connected to: '{0}'.", this.uri));
+
         }
 
         public void Close()
         {
-            logger.Info(string.Format("Closing connection to AltTester® Server on: '{0}'.", this.uri));
+            logger.Info(string.Format("Closing connection to AltTester® on: '{0}'.", this.uri));
```
Comment (andreion1ca) [inline Assets/AltTester/Runtime/AltDriver/Communication/DriverWebSocketClient.cs:197]: Here we should change to `AltTester® Server`?

File: Assets/AltTester/Runtime/AltDriver/Communication/DriverWebSocketClient.cs [hunk 3]
```diff
@@ -117,6 +118,7 @@ protected void OnClose(object sender, CloseEventArgs e)
         {
             logger.Debug("Connection to AltTester® closed: [Code:{0}, Reason:{1}].", e.Code, e.Reason);
```
Comment (andreion1ca) [inline Assets/AltTester/Runtime/AltDriver/Communication/DriverWebSocketClient.cs:119]: Here we should change to `AltTester® Server`?

**Review by andreion1ca — APPROVED** [review]

**Pull request merged**